app/assets/v2/js/vue-components.js

```diff
@@ -701,28 +700,23 @@ Vue.component('suggested-profile', {
     }
   },
   template: `
-<b-media tag="li" class="row mx-auto mx-md-n1">
+<b-media tag="li" class="row mx-auto mx-md-n1 mb-1">
   <template v-slot:aside>
     <a :href="profile_url" class="d-flex nav-link nav-line pr-0 mr-0">
       <b-img :src="avatar_url" class="nav_avatar"></b-img>
     </a>
   </template>
-  <div class="row">
-    <span class="col-6 col-md-12 col-xl-7 font-caption">
+  <div class="col">
+    <span class="row-6 row-md-12 row-xl-7 font-caption">
```

Comment:
Does this work ?

Reply:
Yesss

Reply:
![change](https://user-images.githubusercontent.com/31698676/93105284-80082480-f6cc-11ea-8452-b34f5caa9a7c.png)

cc: @PixelantDesign

Reply:
what exactly are those row- classes?

Reply:
'.row' is a bootstrap grid classe to hold cols, cols are responsive but doesn't make sense to me have row responsive as is not what control sizes.

Reply:
you are using 4.5 booastrap doc, if you notice those row- doesn't exist on the bootstrap version we are using 
https://getbootstrap.com/docs/4.3/layout/grid/ 
If you use the inspector will notice those doesn't do nothing

Reply:
Ohh right! I will make the changes based on v4.3 and push it.